omise/__init__.py

```diff
@@ -1275,11 +1275,11 @@ class Search(_MainResource, Base):
 
         >>> import omise
         >>> omise.api_secret = 'skey_test_4xsjvwfnvb2g0l81sjz'
-        >>> search = omise.Search.execute('charge', {
-            "query": "thb",
-            "filters": {
-                "amount": "1000..2000",
-                "captured": 'true'
+        >>> search = omise.Search.execute('charge', **{
+            'query': 'thb',
+            'filters': {
+                'amount': '1000..2000',
+                'captured': 'true'
```

Comment:
@guzzilar we should because right now code sample and fixture is for old Omise API version, there are many places we need to change for new API version, if we want to change it I think we should do it in another PR.

Reply:
@bank32 ic. thanks for the answer 😄